pythonkr_backend/pythonkr_backend/settings/prod.py

```
@@ -66,4 +66,14 @@
\ No newline at end of file
+#logfire.instrument_psycopg('psycopg')
+
+# celery 
+CELERY_BROKER_PASSWORD = os.environ.get("CELERY_PASSWORD","FALSE")
+CELERY_BROKER_USERNAME = os.environ.get("CELERY_USERNAME","FALSE")
+CELERY_BROKER_VHOST = os.environ.get("CELERY_VHOST","FALSE")
+# Celery Configuration Options
+CELERY_TIMEZONE = "Asia/Seoul"
+CELERY_TASK_TRACK_STARTED = True
+CELERY_BROKER_URL = "amqp://userid:password@localhost:port/virtual_host"
+CELERY_TASK_TIME_LIMIT = 30 * 60
```

Pylint (3.3.7)

[convention] Final newline missing

(C0304)